rust_snuba/src/processors/outcomes.rs

```diff
@@ -0,0 +1,118 @@
+use std::collections::HashSet;
+
+use crate::types::{BadMessage, BytesInsertBatch, KafkaMessageMetadata};
+use rust_arroyo::backends::kafka::types::KafkaPayload;
+use serde::Deserialize;
+
+pub fn process_message(
+    payload: KafkaPayload,
+    _metadata: KafkaMessageMetadata,
+) -> Result<BytesInsertBatch, BadMessage> {
+    let payload_bytes = payload.payload.ok_or(BadMessage)?;
+
+    Ok(BytesInsertBatch { rows: vec![] })
+}
+
+#[derive(Debug, Default, Deserialize)]
+struct FromOutcomeMessage {
+    timestamp: String,
+
+    org_id: Option<u64>,
+    project_id: Option<u64>,
+    key_id: Option<u64>,
+    outcome: u8,
+
+    reason: Option<String>,
+    event_id: Option<String>,
+    category: Option<u8>,
+    quantity: Option<u64>,
```

Comment:
Please check the types. This seems to be a u32 in the database schema: https://github.com/getsentry/snuba/blob/master/snuba/datasets/configuration/outcomes_raw/storages/raw.yaml#L22